### ISSUE TITLE
Exception about reduction-factor and guided-attention

### ISSUE THREAD

**Issue by potato-inoue** [espnet/espnet#1086]
Exception happened when I set `reduction-factor:2` or `reduction-factor:3` with `use-guided-attn-loss: true`.
Also I check the scripts can run in below condition:
- `reduction-factor:1` with `use-guided-attn-loss: true`
- `reduction-factor:2` with `use-guided-attn-loss: false`

Log about the exception is indicated below:
```
[JException in main training loop: The size of tensor a (864) must match the size of tensor b (432) at non-singleton dimension 1
Traceback (most recent call last):
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/chainer/training/trainer.py", line 315, in run
    update()
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/tts/pytorch_backend/tts.py", line 153, in update
    self.update_core()
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/tts/pytorch_backend/tts.py", line 134, in update_core
    loss = self.model(**x).mean() / self.accum_grad
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 489, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py", line 529, in forward
    attn_loss = self.attn_loss(att_ws, ilens, olens)
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 489, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py", line 68, in forward
    losses = self.guided_attn_masks * att_ws
Will finalize trainer extensions and updater before reraising the exception.
Traceback (most recent call last):
  File "/home/abelab/k_inoue/work/tool/espnet/egs/blizzard17/tts1/../../../espnet/bin/tts_train.py", line 179, in <module>
    main(sys.argv[1:])
  File "/home/abelab/k_inoue/work/tool/espnet/egs/blizzard17/tts1/../../../espnet/bin/tts_train.py", line 173, in main
    train(args)
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/tts/pytorch_backend/tts.py", line 424, in train
    trainer.run()
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/chainer/training/trainer.py", line 329, in run
    six.reraise(*sys.exc_info())
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/six.py", line 693, in reraise
    raise value
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/chainer/training/trainer.py", line 315, in run
    update()
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/tts/pytorch_backend/tts.py", line 153, in update
    self.update_core()
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/tts/pytorch_backend/tts.py", line 134, in update_core
    loss = self.model(**x).mean() / self.accum_grad
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 489, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py", line 529, in forward
    attn_loss = self.attn_loss(att_ws, ilens, olens)
  File "/home/abelab/k_inoue/work/tool/espnet/tools/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 489, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/abelab/k_inoue/work/tool/espnet/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py", line 68, in forward
    losses = self.guided_attn_masks * att_ws
RuntimeError: The size of tensor a (864) must match the size of tensor b (432) at non-singleton dimension 1
```

**Comment by kan-bayashi** [comment]
Thank you for your report! I will check it.

**Comment by kan-bayashi** [comment]
Fixed in #1087
